core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -629,9 +639,12 @@ class QualificationAppInfo(
       val totalTransitionsTime = allStagesSummary.map(s => s.transitionTime).sum
       val unsupportedSQLTaskDuration = calculateSQLUnsupportedTaskDuration(allStagesSummary)
       val endDurationEstimated = this.appEndTime.isEmpty && appDuration > 0
-      val jobOverheadTime = calculateJobOverHeadTime(info.startTime)
+      val endTimeJob = info.endTime.getOrElse(0L)
+      val jobOverheadTime = calculateJobOverHeadTime(info.startTime, endTimeJob)
       val nonSQLDataframeTaskDuration =
         calculateNonSQLTaskDataframeDuration(sqlDataframeTaskDuration, totalTransitionsTime)
+
+      // this is weird we are adding wallclock with serialized task duration
```

Comment:
yes, I agree with the comment. Should we postpone the summation until we find a ratio between SQLDFDuration and WallaClock?

Reply:
currently this is not used in calculation rather it it used in output files. @tgravescs mentioned we can keep it as it for now and look into it if we need to pass that argument in - https://github.com/NVIDIA/spark-rapids-tools/issues/979

Reply:
Does this mean that we are not including NonSqlTaskDurations in the estimated ratio?

Reply:
We are not including nonSQLtaskDurations itself in calculation as the wallclock duration for this is implicit.
`appDuration - sqlDfWallckockDurationToUse` should include jobOverheaddTime and nonSQLWallclockDuration

Reply:
Please file an issue to fix and update the comment to point to that issue

Reply:
Thanks! Filed issue - https://github.com/NVIDIA/spark-rapids-tools/issues/987